LeetcodeProblems/Algorithms/2428.Hour_Glass_Sum.js

```diff
@@ -0,0 +1,33 @@
+/*
+2428. Maximum Sum of an Hourglass
+
+You are given an m x n integer matrix grid.
+We define an hourglass as a part of the matrix with the following form:
+Return the maximum sum of the elements of an hourglass.
+
+Note that an hourglass cannot be rotated and must be entirely contained within the matrix.
+Input: grid = [[6,2,1,3],[4,2,1,5],[9,2,8,7],[4,1,2,9]]
+Output: 30
```

Comment:
This could use an explanation of why it's 30.

Reply:
Update the code @ignacio-chiazzo Added as per the format in PR https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/pull/39.

Reply:
Can you please review again?